logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultPathAssertions.kt

```diff
@@ -194,4 +193,31 @@ class DefaultPathAssertions : PathAssertions {
                     container.resolve(entry).collect { _logicAppend { exists(NOFOLLOW_LINKS) } }
                 }
         ).build()
+
+    override fun <T : Path> isEmptyDirectory(container: AssertionContainer<T>): Assertion =
+        if (container.isDirectory().holds()) {
+            container.changeSubject.unreported {
+                it.runCatchingIo { Files.list(it).findAny() }
```

Comment:
I think it would be more helpful to list all entries in the directory. This list can, of course, be pretty long, so maybe we should limit it to some reasonable number?